internal/cow/cow.go

```diff
@@ -88,4 +89,7 @@ type Container interface {
 	Wait() error
 	// Modify sends a request to modify container resources
 	Modify(ctx context.Context, config interface{}) error
+	// Notifications returns a MessageQueue of `notifications.Notification` raised
```

Comment:
*notifications.Message, right?

Reply:
it should be `notifications.Message`, updating now. but if `notifications.Message` is an interface, should it be `*notifications.Message`?